stemcell_builder/stages/system_grub/apply.sh

```diff
@@ -9,10 +9,10 @@ source $base_dir/lib/prelude_apply.bash
 
 # Install grub or grub2 (check existence of classic grub package first because Ubuntu Trusty has a transitional grub2 dummy package)
 
-if pkg_exists grub; then
-  pkg_mgr install grub
-elif pkg_exists grub2; then
+if pkg_exists grub2; then
```

Comment:
Okay I can revert this change. Previous version of Photon used to come with grub and grub2 packages. To avoid installing grub package I have changed the order. As latest version TP2 of Photon only comes with grub2, so it should not affect building Photon stemcell.